osu.Game/Screens/Play/Player.cs

```diff
@@ -248,7 +248,8 @@ private void initializeStoryboard(bool asyncLoad)
             storyboard = beatmap.Storyboard.CreateDrawable(Beatmap.Value);
             storyboard.Masking = true;
 
-            storyboardContainer.Add(asyncLoad ? new AsyncLoadWrapper(storyboard) { RelativeSizeAxes = Axes.Both } : (Drawable)storyboard);
+            if (asyncLoad) LoadComponentAsync(storyboard, s => storyboardContainer.Add(s));
+            else storyboardContainer.Add((Drawable)storyboard);

```

Comment:
cast likely not required.

Reply:
I was thinking about removing it, but kept it in due to it being there in the old version aswell.. Will remove it then